YAML Editor with forwardRef (#11997)
openshift/console: frontend/packages/console-dynamic-plugin-sdk/docs/api.md

| `toolbarLinks` | Array of ReactNode rendered on the toolbar links section on top of the editor. |
| `onChange` | Callback for on code change event. |
| `onSave` | Callback called when the command CTRL / CMD + S is triggered. |

> | `ref` | React reference to monaco editor. For more details, please, visit https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneCodeEditor.html. |

Reviewer: Can we describe the structure of the object seeing as it is `{ editor?: IStandaloneCodeEditor}`

Author: The structure is a little complex. There are, like, hundreds of methods. Do you have any suggestions on how should I describe the structure? Maybe I can say that it can be used to call methods to control the editor?

Reviewer: I wanted to describe that `ref` will have the shape: `{editor?: ...}`. It's not important to describe `IStandaloneCodeEditor` here. The link can remain.

> | `ref` | React reference to `{ editor?: IStandaloneCodeEditor }`. Using the property 'editor' you can have access to all methods to control the editor. For more details, please, visit https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneCodeEditor.html. |

Reviewer 2: ```suggestion
| `ref` | React reference to `{ editor?: IStandaloneCodeEditor }`. Using the 'editor' property, you are able to access to all methods to control the editor. For more information, visit https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneCodeEditor.html. |
```

Author: Thanks @opayne1 . Fixed
